cleaner/docker/cleaner.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2022.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package docker
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/sirupsen/logrus"
+)
+
+// https://docs.docker.com/engine/api/latest/
+func GetClient() (*client.Client, error) {
+	return client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+}
+
+func GetClientRemoteFromEnv(host string) (*client.Client, error) {
+	return client.NewClientWithOpts(client.FromEnv, client.WithHost(host))
+}
+
+func GetClientRemote(host string, cacertPath string, certPath string, keyPath string) (*client.Client, error) {
+	return client.NewClientWithOpts(client.WithHost(host), client.WithAPIVersionNegotiation(), client.WithTLSClientConfig(cacertPath, certPath, keyPath))
+}
+
+func imagesWithCli(cli *client.Client) []types.ImageSummary {
+	images, err := cli.ImageList(context.Background(), types.ImageListOptions{All: true})
+	if err != nil {
+		logrus.Errorf("%s \n", err)
+	}
+
+	return images
+}
+
+func imagesFiltered(cli *client.Client, args filters.Args) []types.ImageSummary {
+	images, err := cli.ImageList(context.Background(), types.ImageListOptions{Filters: args})
+	if err != nil {
+		logrus.Errorf("%s \n", err)
+	}
+	return images
+}
+
+// Remove the images with tag <none>:<none>
```

Comment:
You can add these comments via:

```suggestion
// RemoveImagesUntagged  Remove the images with tag <none>:<none>
```